packages/fury-adapter-remote/package.json

```diff
@@ -23,13 +23,13 @@
   },
   "dependencies": {
     "axios": "^0.19.0",
+    "chai": "^4.2.0",
```

Comment:
I think this change was accidental? chai is a devDependencies as it's only needed for the tests.

Reply:
I received a warning somewhere I guess; changed